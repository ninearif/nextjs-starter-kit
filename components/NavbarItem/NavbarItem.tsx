type Props = {};

const NavbarItem = ({}: Props) => {
  return <div>NavbarItem</div>;
};

export default NavbarItem;
