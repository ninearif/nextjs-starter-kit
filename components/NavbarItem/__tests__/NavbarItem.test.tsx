import { render, screen } from '@testing-library/react';

import NavbarItem from '../NavbarItem';

describe('NavbarItem component test', () => {
  it('NavbarItem is rendered as expected.', () => {
    render(<NavbarItem />);

    expect(screen.getByText('NavbarItem')).toBeInTheDocument();
    // expect(screen.getByRole("heading")).toHaveTextContent(/NavbarItem/);
    // expect(screen.getByRole("button", { name: "submit" })).toBeDisabled();
    // expect(screen.getByRole("img")).toBeInTheDocument();
    // some cheat sheet can be found here:
    // https://testing-library.com/docs/react-testing-library/cheatsheet/
  });
});

describe('NavbarItem component snapshot test', () => {
  it('render NavbarItem unchanged', () => {
    const { container } = render(<NavbarItem />);
    expect(container).toMatchSnapshot();
  });
});
