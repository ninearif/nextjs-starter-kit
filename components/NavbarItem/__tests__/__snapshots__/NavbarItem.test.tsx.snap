// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarItem component snapshot test render NavbarItem unchanged 1`] = `
<div>
  <div>
    NavbarItem
  </div>
</div>
`;
